Answers.md

```diff
@@ -0,0 +1,21 @@
+Describe the biggest difference between .forEach & .map.
+
+Answer= forEach calls back each element in an array, map returns a new array that is the same size.
+
+What is the difference between a function and a method?
+
+Answer:A methord is a fuction that happens to be the property of an object.
+
+What is closure?
+
+Answer: a way to make Javascript fuctions use there own Variables.
```

Comment:
Your original answer was a lot closer, think about scope and how it relates to global and local variables